src/main/java/teamrtg/highlandsut/References.java

```diff
@@ -4,6 +4,6 @@
 
     public static final String MOD_ID = "highlandsut";
     public static final String MOD_NAME = "Highlands UT";
-    public static final String MOD_VERSION = "0.0.1";
+    public static final String MOD_VERSION = "0.1.0";
```

Comment:
I was going to PR a fix for this to go back to the original setup of @substitutions@ from the build scripts, if you don't mind.